docx_form/content_controls/OptionalContentControl.py

```diff
@@ -0,0 +1,152 @@
+        self.display_text = display_text
+        self.value = value
+
+
+class OptionalContentControl(DocxContentControl):
+    """
+    This class contains all properties and methods for an optional content control.
+
+    :param DocxContentControl: This class extends the DocxContentControl class
+    """
+
+    def __init__(self, root: Element, file_path: str):
+        super().__init__(root, file_path)
+
+    def _DropDownListContentControl__get_options(self) -> list[Option] | None:
+        """
+        This method gets the options of a optional content control.
+
+        :return: The options of the optional content control
+        :rtype: list[Option] | None
+        """
+        # The root of the document
+        root: Element = etree.XML(Raw_XML.raw_xml)
+        # The body of the document
+        body: Element = root.getchildren()[0]
+
+        element: Element
+        for element in body.iter(f"{XML_PREFIX}sdt"):
+            # Get the nested id tag
+            id_tag: Element
+            for id_tag in element.iter(f"{XML_PREFIX}id"):
+                # If the id matches
+                if id_tag.get(f"{XML_PREFIX}val") == self.id:
+                    # Get the options
+                    options: list[Option] = []
+                    option = element
+                    for option in element.iter(f"{XML_PREFIX}listItem"):
+                        tag_info = Option(
+                            option.get(f"{XML_PREFIX}displayText"),
+                            option.get(f"{XML_PREFIX}value"),
+                        )
+                        options.append(tag_info)
+
+                    return options
+
+        return None
+
+    def _ComboBoxContentControl__get_options(self) -> list[Option] | None:
+        """
+        This method gets the options of a optional content control.
+
+        :return: The options of the optional content control
+        :rtype: list[Option] | None
+        """
+        # The root of the document
+        root: Element = etree.XML(Raw_XML.raw_xml)
+        # The body of the document
+        body: Element = root.getchildren()[0]
+
+        element: Element
+        for element in body.iter(f"{XML_PREFIX}sdt"):
+            # Get the nested id tag
+            id_tag: Element
+            for id_tag in element.iter(f"{XML_PREFIX}id"):
+                # If the id matches
+                if id_tag.get(f"{XML_PREFIX}val") == self.id:
+                    # Get the options
+                    options: list[Option] = []
+                    option = element
+                    for option in element.iter(f"{XML_PREFIX}listItem"):
+                        tag_info = Option(
+                            option.get(f"{XML_PREFIX}displayText"),
+                            option.get(f"{XML_PREFIX}value"),
+                        )
+                        options.append(tag_info)
+
+                    return options
+
+        return None
```

Comment:
Did you mean to make these separate? It seems like you just call __get_options() in each subclass.